docs/user-guide/zwe-init-subcommand-overview.md

```diff
@@ -68,37 +59,31 @@ The following output is an example of running `zwe init mvs`.
 -------------------------------------------------------------------------------
 >> Initialize Zowe custom data sets
 
-Create data sets if they are not exist
+Create data sets if they do not exist
 Creating IBMUSER.ZWE.CUST.PARMLIB
 Creating IBMUSER.ZWE.CUST.JCLLIB
-Creating IBMUSER.ZWE.SZWEAUTH
+Creating IBMUSER.ZWE.CUST.ZWESALL
```

Comment:
Should this be reverted to `SZWEAUTH`? There are many more references to `SZWEAUTH` throughout the Zowe doc

Reply:
1) My impression of example is to demonstrate creation of the custom libraries.
2) The master version is weird, we are creating `IBMUSER.ZWE.SZWEAUTH` there, which is normally created by `zwe install`.
https://github.com/zowe/docs-site/blob/ac7e23f55e1f1e9eaa69f5383277fdc8fe58e1b1/docs/user-guide/zwe-init-subcommand-overview.md?plain=1#L72-L75

Reply:
addressed